src/css/application.css

```diff
@@ -194,6 +194,18 @@ body {
   transform-origin: right 50%;
 }
 
+.sidebar__component {
+  text-align: center;
+  padding-bottom: 1rem;
+  font-size: 1.5rem;
+}
+
+.sidebar__plusMinus {
+  font-family: 'FontAwesome';
+  position: relative;
+  margin: auto;
+}
```

Comment:
I would factor the CSS thusly:

* Create a `u__icon` utility element that applies the FontAwesome font, as well as a `cursor: pointer` and `user-select: none`. Use it here and also apply it to the other FontAwesome icons in the UI.
* Combine the `sidebar__component` and `sidebar__plusMinus` elements into a single element (keep the `sidebar__plusMinus` name)

Seem reasonable?